tests/cnf/core/network/metallb/tests/bgp-remote-as-dynamic.go

```diff
@@ -22,9 +23,12 @@ import (
 var _ = Describe("BGP remote-dynamicAS", Ordered, Label(tsparams.LabelDynamicRemoteASTestCases),
 	ContinueOnFailure, func() {
 		var (
-			err           error
-			dynamicASiBGP = "internal"
-			dynamicASeBGP = "external"
+			err                        error
+			dynamicASiBGP              = "internal"
+			dynamicASeBGP              = "external"
+			hubPodWorker0              = "hub-pod-worker-0"
```

Comment:
Please move it under multihop context.

Reply:
Moved both
			hubPodWorker0              = "hub-pod-worker-0"
			hubPodWorker1              = "hub-pod-worker-1"